ui/components/ui/button-group/button-group.component.js

```diff
@@ -4,13 +4,37 @@ import classnames from 'classnames';
 
 export default class ButtonGroup extends PureComponent {
   static propTypes = {
+    /**
+     * change button active order
+     */
     defaultActiveButtonIndex: PropTypes.number,
+    /**
+     * no button are active before clicked by the user
+     */
     noButtonActiveByDefault: PropTypes.bool,
+    /**
+     * disabling every button inside button group
+     */
     disabled: PropTypes.bool,
+    /**
+     * Children must be an array of button components
+     */
     children: PropTypes.array,
+    /**
+     * classname is used for children prop, changing it in storybook won't change anything
```

Comment:
`className` seems to add a class to the parent component? Is that not what you've experienced? Also not sure we should mention storybook anymore as it's inside of the component

```suggestion
     * Adds a className to the root div of the of the ButtonGroup component
```

<img width="1438" alt="Screen Shot 2021-11-10 at 12 02 42 PM" src="https://user-images.githubusercontent.com/8112138/141185206-ca83859b-98db-4b58-96d0-19b83312d950.png">